client/index.html

```diff
@@ -58,6 +58,7 @@ <h1 class="title">The Lounge is loading…</h1>
 				<div id="chat-container" class="window">
 					<div id="chat"></div>
 					<form id="form" method="post" action="">
+						<div id="connection-error"><span>Connection failed: <span id="connection-error-reason">??</span>. Refresh the page to reconnect.</span></div>
```

Comment:
> Outer <span> is an accident here?

Nope. It's to correctly do the flexbox.

> Also, I don't think there is any need for specific message. 

Specific messages absolutely help when you hit a certain issue.